src/sentry/issues/status_change.py

```diff
@@ -91,6 +91,7 @@ def handle_status_update(
     for group in group_list:
         group.status = new_status
         group.substatus = new_substatus
+        group.save()
```

Comment:
Hmm, how did this work before without us calling `save()`??

Reply:
Ugh, so looks like we're using `queryset.update()` just before `handle_status_update` to bulk update the group `status` and `substatus`. `update()` doesn't trigger `pre_save()` at all and bypasses any validation we can't rely on there. We probably have to sift through any uses of update and apply some adhoc validation/defaulting.

Reply:
> update() doesn't trigger pre_save()  

oh yikes. Is this something we can change so the `pre_save` signal always fires on `update()`?

Reply:
None that I could think of. `update()` is basically a way to bypass django and it generates raw sql to bulk update multiple rows in one shot.  

It might be ok if we swap the single update call for multiple saves:
```
for group in grouplist:
    group.save(update_fields=["status", "substatus"])
```
it'll trigger multiple postgres calls but maybe its ok depending on how many groups get passed in to `update_groups`.

The alternative is we keep using update and just create a function that we have to call before update to validate the `status` and `substatus` fields.

Reply:
> swap the single update call for multiple saves

This sounds alright to me. Let's take note of endpoints that batch handle groups and setup a dashboard to keep an eye on the p95